lib/api/src/modules/refs.ts

```diff
@@ -168,7 +164,7 @@ export const init: ModuleFn = ({ store, provider, fullAPI }, { runCheck = true }
         return {};
       };
 
-      if (!included && !omitted && !iframe) {
+      if (!included && !omitted && type !== 'server-checked') {
```

Comment:
I would expect to see a test for this scenario.

Reply:
added a test!